pemFioi/quickAlgo/interface-mobileFirst.js

```diff
@@ -302,7 +308,78 @@ var quickAlgoInterface = {
             this.displayBlocklyPython();
         } else if (btn == 'about') {
             this.openAbout();
+        } else if (btn == 'share') {
+            this.openShare();
+        }
+    },
+
+    openShare: function() {
+        displayHelper.showPopupMessage(this.strings.doYouWantToShare, 'blanket', this.strings.yes, this.shareSubject, this.strings.no, null, null)
```

Comment:
```suggestion
        displayHelper.showPopupMessage(this.strings.doYouWantToShare, 'blanket', this.strings.yes, this.shareSubject.bind(this), this.strings.no, null, null)
```

This will avoid the issue you describe below at line 321.

Reply:
Added the bind with this, and removed the issue at line 321